src/ray/pubsub/subscriber.cc

```diff
@@ -111,13 +113,13 @@ void SubscriberChannel<KeyIdType>::HandlePublisherFailure(
   auto &subscription_callback_map = subscription_it->second.subscription_callback_map;
 
   std::vector<KeyIdType> key_ids_to_unsubscribe;
+  std::shared_ptr<SubscriptionFailureCallback> failure_callback;
```

Comment:
Why do you need this? I see later we have another failure_callback defined in the for loop.

Reply:
Will remove.